tests/test_tasks.py

```diff
@@ -53,6 +53,23 @@ def test_copy_tasks(template_path: str, tmp_path: Path) -> None:
     assert (tmp_path / "pyfile").is_file()
 
 
+def test_copy_skip_tasks(template_path: str, tmp_path: Path) -> None:
+    copier.run_copy(
+        template_path,
+        tmp_path,
+        quiet=True,
+        defaults=True,
+        overwrite=True,
+        unsafe=True,
```

Comment:
This test should also pass with

```suggestion
        unsafe=False,
```

right? I think we should test it that way to cover that `skip_tasks=True` makes `unsafe=True` unnecessary (when no other unsafe features are used – like here).

Reply:
Yes it should but I don't see the point given that we already test that `unsafe=False` disable extensions, tasks and migrations.
That's exactly the point of this test: check that only tasks are skipped when the template is trusted and `--skip-tasks` is enabled. 
This change would just test that `--skip-tasks` don't do anything when the template is not trusted (eg. when tasks are already skipped).
The need to skip tasks only exists for trusted templates (eg. with extensions, tasks and migrations enabled), more precisely the need is to be able to run a template with extensions active but without the running tasks or to update with the extensions and migrations active but without the tasks. 
(Note that I took into account that it is planned that trusts will be persistent at some point, so this test should still pass when it is the case)

Reply:
Edit: I just tried to parametrize `unsafe`, but it doesn't run for `unsafe=False`: trying to run a template with `tasks` and without `unsafe` raise `UnsafeTemplateError` (as expected by https://github.com/copier-org/copier/blob/master/tests/test_unsafe.py in fact)

Reply:
Ah, I missed the rationale you explained in https://github.com/copier-org/copier/pull/1561#issuecomment-2015137696. With that in mind, the behavior you observe is expected.

I wonder whether `--skip-tasks` should actually affect the `--trust` behavior. When you pass `--skip-tasks`, you avoid the security risk of tasks. So why should `copier copy --skip-tasks ...` fail because of unsafe features when the only unsafe feature of that template is "tasks"? I'm not sure that's intuitive. WDYT, @copier-org/maintainers?

Reply:
@noirbizarre recalled previously that not only tasks are unsafe, but Jinja extensions too :slightly_smiling_face: 
Therefore `--skip-tasks` **should not** silently toggle `--trust` on.

Reply:
Yes, I didn't mean to toggle `--trust` but only ignore tasks when determining the unsafe features of a template when `--skip-trust` is provided.

I think you've convinced me to leave behavior as it is now. Also, we won't have `--skip-{jinja-extensions,migrations}` because skipping those would render the template useless. So considering `--skip-tasks` as a partial `--trust` seems conceptually wrong – adding to your reasoning towards the same conclusion.